### ISSUE TITLE
[Bug][/sql/dolphinscheduler-postgre.sql][1.3.2-release] No "state" column in t_ds_user table

### ISSUE THREAD

**Issue by huiyuanlu** [apache/incubator-dolphinscheduler#4046]
**Describe the bug**
/sql/dolphinscheduler-postgre.sql line 786
There is no "state" column in t_ds_user table, when insert data.

**Expected behavior**
The correct sql is 
```sql
INSERT INTO t_ds_user(user_name,user_password,user_type,email,phone,tenant_id,create_time,update_time) VALUES ('admin', '7ad2410b2f4c074479a8937a28a22b8f', '0', 'xxx@qq.com', '', '0', '2018-03-27 15:48:50', '2018-10-24 17:40:22');
```

**Which version of Dolphin Scheduler:**
 -[1.3.2-release]

**Additional context**

```sql
--line 615 - 628 
DROP TABLE IF EXISTS t_ds_user;
CREATE TABLE t_ds_user (
  id int NOT NULL  ,
  user_name varchar(64) DEFAULT NULL ,
  user_password varchar(64) DEFAULT NULL ,
  user_type int DEFAULT NULL ,
  email varchar(64) DEFAULT NULL ,
  phone varchar(11) DEFAULT NULL ,
  tenant_id int DEFAULT NULL ,
  create_time timestamp DEFAULT NULL ,
  update_time timestamp DEFAULT NULL ,
  queue varchar(64) DEFAULT NULL ,
  PRIMARY KEY (id)
);

-- line 786
INSERT INTO t_ds_user(user_name,user_password,user_type,email,phone,tenant_id,state,create_time,update_time) VALUES ('admin', '7ad2410b2f4c074479a8937a28a22b8f', '0', 'xxx@qq.com', '', '0', 1, '2018-03-27 15:48:50', '2018-10-24 17:40:22');
```

**Requirement or improvement**

```

**Comment by xingchun-chen** [comment]
There is no state field in version 1.3.2, currently only the dev branch has the state field

**Comment by huiyuanlu** [comment]
Thanks for your reply, I know we have add state field in version 1.3.3-release and dev branch, but not in version 1.3.2-release, right?

I find this question when I download "Binary Distribution" from dolphinscheduler.apache.org which version is 1.3.2. So I checked the 1.3.2-realse branch in github, the SQL is not matched.

This question is easy to find, I marked the question in the issues when someone meet this.
